project-koku/koku
# Fixing OCP/GCP project credit distribution

## Jira Ticket

[COST-2453](https://issues.redhat.com/browse/COST-2453)

## Description

This change will fix OCP on GCP project level credit distribution

## Testing

1. Checkout Branch
2. Restart Koku
3. Load OCP on GCP data with multiple projects that have matching credits
4. See we have credits distributed correctly based from projects usage via api/cost-management/v1/reports/openshift/infrastructures/gcp/costs/?group_by[project]=*

## Notes

...
